.github/workflows/backend-code.yml

```
@@ -1,4 +1,4 @@
-name: Code
+name: backend-code
```

This doesn't need to be machine readable
```suggestion
name: Backend Code
```